EveripediaNetwork/ep-ui
# Wiki Flickering Fix & Arrangement

# Wiki Flickering Fix & Arrangement

_This PR fixes the wikis flickering on profile page and adjusts them to span four columns_

## How should this be tested?

1. Visit https://iq-wiki-git-profile-wiki-prediqt.vercel.app then sign in
2. Navigate to profile
3. Confirm Wikis don't flicker anymore when you scroll
4. Also confirm we now have up to 4 wikis per row (for both Wikis and Edits).

## Notes

_Screen size determines the number of wiki displayed per row_

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3117
